$OpenBSD: patch-docs_source_conf_py,v 1.5 2017/05/01 15:37:15 shadchin Exp $

A problem with inheritance diagrams. Causes build to crash. Even if it
did work, the generated diagrams have non-deterministic names, making
putting their name in a PLIST somewhat difficult.

Disable intersphinx.

--- docs/source/conf.py.orig	Fri Feb 24 20:12:01 2017
+++ docs/source/conf.py	Sun Apr 23 20:51:07 2017
@@ -55,8 +55,6 @@ extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
     'sphinx.ext.doctest',
-    'sphinx.ext.inheritance_diagram',
-    'sphinx.ext.intersphinx',
     'IPython.sphinxext.ipython_console_highlighting',
     'IPython.sphinxext.ipython_directive',
     'sphinx.ext.napoleon',  # to preprocess docstrings
